wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/MessageAction.swift

```diff
@@ -21,7 +21,8 @@ import WireDataModel
 import UIKit
 
 enum MessageAction: CaseIterable, Equatable {
-    static var allCases: [MessageAction] = [.digitallySign,
+    static var allCases: [MessageAction] = [.visitLink("https://wire.com"),
```

Comment:
Is this just because you need to provide some link for the all cases? Or do we actually use this link?

Reply:
It is just because we need to provide some link. This is not used.